admin_tests/base/test_utils.py

```diff
@@ -158,3 +165,48 @@ def test_change_embargo_date(self):
         assert_almost_equal(self.registration.embargo.end_date, self.date_valid2, delta=datetime.timedelta(days=1))
 
         # Add a test to check privatizing
+
+site = AdminSite()
+
+class TestGroupCollectionsPreprints(AdminTestCase):
+
+    @pytest.mark.enable_bookmark_creation
+    def test_admin_app_groups(self):
+        user = UserFactory()
+        user.save()
+        request = RequestFactory()
+        get_request = request.get('/admin/osf/osfuser/{}/change'.format(user.id))
+        get_request.user = user
+        post_request = request.post('/admin/osf/osfuser/{}/change'.format(user.id))
```

Comment:
DRY up defining this URL so you can use it for both the get_request and the post_request!